cloudmarker/clouds/gcpcloud.py

```diff
@@ -233,35 +242,50 @@ def _make_record(self, iterator, gcp_record_type, project_index, project,
         record_type_map = {
             'compute': 'compute',
         }
-        for i, raw_record in enumerate(iterator):
-            record = {
-                'raw': raw_record,
-                'ext': {
-                    'cloud_type': 'gcp',
-                    'record_type': gcp_record_type,
-                    'project_id': project.get('projectId'),
-                    'project_name': project.get('name'),
-                    'zone': zone,
-                    'key_file_path': self._key_file_path,
-                    'client_email': self._client_email
-                },
-                'com': {
-                    'cloud_type': 'gcp',
-                    'record_type': record_type_map.get(gcp_record_type)
+
+        try:
+            for i, raw_record in enumerate(iterator):
+                record = {
+                    'raw': raw_record,
+                    'ext': {
+                        'cloud_type': 'gcp',
+                        'record_type': gcp_record_type,
+                        'project_id': project.get('projectId'),
+                        'project_name': project.get('name'),
+                        'zone': zone,
+                        'key_file_path': self._key_file_path,
+                        'client_email': self._client_email
+                    },
+                    'com': {
+                        'cloud_type': 'gcp',
+                        'record_type': record_type_map.get(gcp_record_type)
+                    }
                 }
-            }
 
-            _log.info('Found %s #%d: %s; %s', gcp_record_type, i,
-                      raw_record.get('name'),
-                      util.outline_gcp_project(project_index, project, zone,
-                                               self._key_file_path))
-            yield record
+                _log.info('Found %s #%d: %s; %s', gcp_record_type, i,
+                          raw_record,
```

Comment:
I think there is no need to log the whole `raw_record`. Just `raw_record.get('name')` should be enough, as it was before. 
```suggestion
                          raw_record.get('name'),
```